### DISCUSSION TITLE
Invalid password - escaping issue?

### DISCUSSION THREAD

**Discussion by ash2k** [AdguardTeam/AdGuardHome#3387] [Q&A]
### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.3
* **How did you install AdGuard Home:**
  * Docker
* **How did you setup DNS configuration:**
  * Router
* **If it's a router or IoT, please write device model:**
  * N/A
* **CPU architecture:**
  * arm64
* **Operating system and version:**
  * N/A

### Expected Behavior

I use a password generator and it gave me a password with a `%` sign in it. I expect it to work fine OR reject passwords with unsupported characters.

### Actual Behavior

Couldn't log in after installation was finished. Re-installed and used the same password without the `%` sign - all works fine now. Missing escaping somewhere?

**Comment by ainar-g** [comment]
Hello.  We cannot reproduce this issue, at least on the current `edge`.  What browser are you using?  Are you sure that the password was correct the first time (e.g. there was no accidental keyboard layout switch)?

**Comment by ash2k** [comment]
I'm on latest Firefox. I generated the password and copied it, I didn't type it. Hm, maybe I thought I copied it but didn't? If you cannot reproduce then I'm not sure what else I can help with :) It was probably my own fault then. Thanks for a great piece of software!
